python/xrprimer/visualization/matplotlib/plot_frame.py

```diff
@@ -0,0 +1,192 @@
+import io
+from typing import Union
+
+import cv2
+import numpy as np
+
+from xrprimer.utils.log_utils import get_logger, logging
+from xrprimer.visualization.palette.line_palette import LinePalette
+from xrprimer.visualization.palette.point_palette import PointPalette
+
+try:
+    from matplotlib import pyplot as plt
+    from mpl_toolkits.mplot3d import Axes3D
+    has_matplotlib = True
+    import_exception = ''
+except (ImportError, ModuleNotFoundError):
+    has_matplotlib = False
+    import traceback
+    stack_str = ''
+    for line in traceback.format_stack():
+        if 'frozen' not in line:
+            stack_str += line + '\n'
+    import_exception = traceback.format_exc() + '\n'
+    import_exception = stack_str + import_exception
+
+
+def plot_frame(point_palette: Union[PointPalette, None] = None,
```

Comment:
What about renaming it to `plot_3d_frame`?

Reply:
Hard to tell that whether a frame plotted by opencv is not a 3D frame. We will also have babylon or mpr in the future, and they will also share the same name like matplotlib.
Considering the main difference, I think the difference lies only in the third-party libraries that are called. Therefore, naming conventions may or may not reflect information about the third-party libraries, but not other information.
Using the same name can also help users understand that these functions with the same name sahre the same 
operation.